lib/src/code/code.dart

```diff
@@ -26,15 +38,15 @@ class Code {
 
   static List<CodeLine> _textToCodeLines(
     String text,
-    List<String> singleLineComments,
+    String? singleLineComment,
```

Comment:
Some languages support multiple sequences for single line comments:
https://www.php.net/manual/en/language.basic-syntax.comments.php

So this should still be a list.